gtda/plotting/persistence_diagrams.py

```diff
@@ -63,8 +63,18 @@ def plot_diagram(diagram, homology_dimensions=None, plotly_params=None):
         subdiagram = diagram[diagram[:, 2] == dim]
         diff = (subdiagram[:, 1] != subdiagram[:, 0])
         subdiagram = subdiagram[diff]
-        fig.add_trace(gobj.Scatter(x=subdiagram[:, 0], y=subdiagram[:, 1],
-                                   mode="markers", name=name))
+        unique, inverse, counts = np.unique(
+            subdiagram, axis=0, return_inverse=True, return_counts=True
+            )
+        hovertext = [
+            "<br>".join([f"{tuple(unique[unique_row_index][:2])}"]
```

Comment:
I like the idea of including multiplicity. However, I am worried that this might be unreadable when the multiplicity is >10 (this can be the case for images).

Reply:
What about adding it in the however text if the multiplicity is greater than 1?
For example:
"(2,3)" or "(2,3), multiplicity: 5",

Reply:
A great suggestion!

Reply:
Done.